exercises/ex1.cpp

```diff
@@ -11,37 +11,33 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
+  switch(week){
+    case 1:
+      cout << "Monday" << endl;
+    break;
+
+    case 2:
+      cout << "Tuesday" << endl;
+    break;
+
+    case 3:
+      cout << "Wednesday" << endl;
+    break;
+    case 4:
+      cout << "Thursday" << endl;
+    break;
+    case 5:
+      cout << "Friday" << endl;
+    break;
+    case 6:
+      cout << "Saturday" << endl;
+    break;
+    case 7:
+      cout << "Sunday" << endl;
+    break;
+    default:
+        cout << "Invalid input! Please enter week number between 1-7." << endl;
+    break;
   }
 
   return 0;
```

Comment:
Potresti rendere più esplicito il valore di ritorno del **main** aggiungendo in cima al codice, subito dopo gli "include"

```c++
#define EXIT_SUCCESS 0
```

e alla fine del **main**, al posto di "return 0;"

```c++
return EXIT_SUCCESS;
```